docs/js-core-3/module-systems/lesson.md

```diff
@@ -0,0 +1,399 @@
+
+- The starting location is the `script.js` file inside the `project` folder
+- The destination location will be the `portfolio.html`
+
+The file path will therefore be:
+
+```
+../portfolio.html
+```
+
+This file path looks quite similar, we have some folder or file names separated by slashes. The main difference is the two dots at the start, which also has a special meaning for file paths. a shortcut to the **parent** folder of the starting location.
+
+![File path to portfolio.html. An arrow goes up from script.js to portfolio, then up again to cyf. Then down to portfolio.html](./assets/portfolio-html-file-path.png)
+
+In "plain English", we could say:`
+
+> Starting in the folder of the starting location, move one level "up" the tree, then look for a file named `portfolio.html`
+
+For our final example, we'll say:
+
+- The starting location is the `script.js` file inside the `project` folder
+- The destination location will be the `exercise.js` file inside the `coursework` folder
+
+The file path will therefore be:
+
+```
+../coursework/exercise1.js
+```
+
+Now we're using multiple slashes to move between the `project` folder and the `coursework` folder.
+
+![File path to exercise.js. An arrow goes up from script.js to project, then again up to cyf. The arrow goes down to coursework then down again to exercise.js](./assets/exercise-js-file-path.png)
+
+In "plain English", we could say:
+
+> Starting in the folder of the starting location, move one level "up" the tree, then move into a folder named `coursework`, then look for a file named `exercise.js`
+
+| Exercise A                                                                                                                |
+| ------------------------------------------------------------------------------------------------------------------------- |
+| If the starting location is the `script.js` in the `projects` folder, what is the file path when the destination file is: |
+| 1. `styles.css` in the `projects` folder                                                                                  |
+| 2. `exercise2.js` in the `coursework` folder                                                                              |
+| 3. `week1.txt` in the `js3` folder                                                                                        |
+
+You might start to recognise the file path patterns! You've actually already used file paths when using the `git add` command, although you might not have realised it. File paths are used in many other commands too.
+
+As we looked at earlier, we can also use file paths when importing files in JavaScript. We can now explain what this code does:
+
+```js
+const findWidget = require("./widget-finder");
+```
+
+Now we know that the `./widget-finder` file path refers to a file in the same folder called `widget-finder.js`. We can write out the `.js` bit if we want, but if you leave it off, then the module system will assume that you mean `.js`.
```

Comment:
maybe we should leave the `.js` in the examples (for clarity) - and then teach that it can actually be dropped later, once it's really clear